src/commands/test/lib/calabash-preparer.ts

```diff
@@ -64,28 +64,18 @@ export class CalabashPreparer {
     }
 
     if (this.testParameters && this.testParameters.length > 0) {
-      command += ` --test-params ${this.generateTestParameterArgs()}`;
+      command += this.generateTestParameterArgs();
     }
 
     return command;
   }
 
   private generateTestParameterArgs(): string {
     let result: string = "";
-
-    if (this.testParameters) {
-      this.testParameters.forEach(p => {
-        let parsedParameter = parseTestParameter(p);
-        if (result != "") {
-          result += ",";
```

Comment:
If a single `--test-parameters` is required, the delimiter should be `" "`, not `",".